include/grpc++/impl/codegen/thrift_serializer.h

```diff
@@ -131,12 +131,12 @@ class ThriftSerializer {
     last_deserialized_ = true;
 
     // reset buffer transport
-    buffer_->resetBuffer((uint8_t*)serialized_buffer, length);
+    buffer_->resetBuffer(const_cast<uint8_t*>(serialized_buffer), length);
 
     // read the protocol version if necessary
     if (serialize_version_) {
       std::string name = "";
-      TMessageType mt = (TMessageType)0;
+      TMessageType mt = static_cast<TMessageType>(0);
```

Comment:
Because of this 

> /usr/local/include/grpc++/impl/codegen/thrift_serializer.h:139:24: error: invalid conversion from 'int' to 'apache::thrift::protocol::TMessageType' [-fpermissive]
>        TMessageType mt(0);
>